frontend/src/components/requisitionForm/RequisitionForm.tsx

```diff
@@ -204,7 +263,8 @@ const RequisitionForm: React.FunctionComponent = () => {
         <Row justify="center">
```

Comment:
Would be nice if a live running total cost was shown at the bottom of the page

Reply:
Added